### PR TITLE
docs: add information on how otel collector collects data 

### PR DESCRIPTION
**Pull request by nitishfy**
added information about how otel collector collects telemetry data

### CODE REVIEW

File: data/docs/userguide/send-metrics-cloud.mdx [hunk 1]
```diff
@@ -7,20 +7,90 @@ title: Send Metrics to SigNoz Cloud
...
 - [Enable a Specific Metric Receiver](#enable-a-specific-metric-receiver)
 - [Enable a Prometheus Receiver](#enable-a-prometheus-receiver)
 - [Find Metrics available in SigNoz](#find-metrics-available-in-signoz)
   - [Metrics from Hostmetrics receiver](#metrics-from-hostmetrics-receiver)
 - [Related Videos](#related-videos)
 - [Get Help](#get-help)
 
+## How does OpenTelemetry Collector collect data?
+
+Data collection in OpenTelemetry Collector is facilitated through receivers. Receivers are configured via YAML under the top-level `receivers` tag. To ensure a valid configuration, at least one receiver must be enabled.
```
Comment (Calm-Rock) [inline data/docs/userguide/send-metrics-cloud.mdx]:
```suggestion
Data collection in OpenTelemetry Collector is facilitated through receivers. Receivers are configured via YAML under the top-level `receivers` section. To ensure a valid configuration, at least one receiver must be enabled.
```

[tags](https://www.javatpoint.com/yaml-tags) in YAML is something else.

File: data/docs/userguide/send-metrics-cloud.mdx [hunk 2]
```diff
@@ -7,20 +7,90 @@ title: Send Metrics to SigNoz Cloud
...
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+```
+
+The OTLP receiver accepts data through gRPC or HTTP in the <a href = "https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md" rel="noopener noreferrer nofollow" target="_blank" >OTLP</a> format. There are advanced configurations that you can enable via the YAML file.
```
Comment (Calm-Rock) [inline data/docs/userguide/send-metrics-cloud.mdx]:
```suggestion
The OTLP receiver accepts data through gRPC or HTTP in the <a href = "https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md" rel="noopener noreferrer nofollow" target="_blank" >OTLP</a> format. 
```

File: data/docs/userguide/send-metrics-cloud.mdx [hunk 3]
```diff
@@ -7,20 +7,90 @@ title: Send Metrics to SigNoz Cloud
...
+          allowed_origins:
+            - http://test.com
+            # Origins can have wildcards with *, use * by itself to match any origin.
+            - https://*.example.com
+          allowed_headers:
+            - Example-Header
+          max_age: 7200
+```
+
+You can find more details on advanced configurations <a href = "https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/README.md" rel="noopener noreferrer nofollow" target="_blank" >here</a>.
```
Comment (Calm-Rock) [inline data/docs/userguide/send-metrics-cloud.mdx]: To see more configuration options for otlp receiver, you can checkout [this link](https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/README.md)

File: data/docs/userguide/send-metrics-cloud.mdx [hunk 4]
```diff
@@ -7,20 +7,90 @@ title: Send Metrics to SigNoz Cloud
...
+    metrics:
+      receivers: [otlp, httpcheck]
+      processors: [batch]
+      exporters: [otlp]
+```
+
+Here’s a breakdown of the above metrics pipeline:
+
+- **Receivers:** This pipeline is configured to receive metrics data from two sources: OTLP and HTTP Check. The **`otlp`** receiver collects metrics using both gRPC and HTTP protocols, while the **`httpcheck`** receiver gathers metrics from the HTTP endpoint.
+- **Processors:** Metrics data is processed using the **`batch`** processor. This processor likely batches metrics before exporting them, optimizing the data flow.
```
Comment (Calm-Rock) [inline data/docs/userguide/send-metrics-cloud.mdx:91]:
```suggestion
- **Processors:** Metrics data is processed using the **`batch`** processor. This processor batches metrics before exporting them, optimizing the data flow.
```

File: data/docs/userguide/send-metrics-cloud.mdx [hunk 5]
```diff
@@ -7,20 +7,90 @@ title: Send Metrics to SigNoz Cloud
...
+- **Exporters:** Metrics processed through this pipeline are exported to the OTLP destination. The **`otlp`** exporter sends data to an endpoint specified in the configuration.
```
Comment (Calm-Rock) [inline data/docs/userguide/send-metrics-cloud.mdx:92]:
```suggestion
- **Exporters:** Metrics processed through this pipeline are exported to the OTLP endpoint mentioned in the configuration file. 
```

**Review by Calm-Rock — CHANGES_REQUESTED** [review]

File: data/docs/userguide/send-metrics-cloud.mdx (hunk 1 above)
Comment (nitishfy) [reply data/docs/userguide/send-metrics-cloud.mdx]: gotcha!

**Review by Calm-Rock — APPROVED** [review]

**Pull request merged**